c11threads.h

```diff
@@ -159,14 +170,15 @@ static inline int mtx_trylock(mtx_t *mtx)
 
 static inline int mtx_timedlock(mtx_t *mtx, const struct timespec *ts)
 {
-	int res;
+	int res = 0;
 #ifdef C11THREADS_NO_TIMED_MUTEX
 	/* fake a timedlock by polling trylock in a loop and waiting for a bit */
-	struct timeval now;
-	struct timespec sleeptime;
+	struct timeval now = {0};
+	struct timespec sleeptime = {0};
```

Comment:
I don't see why any of these need to be initialized to zero. Am I missing something?

Reply:
Yeah, no reason... I will remove it (out of habit)

Reply:
Cleaned.